elasticsearch-ilm.html.md.erb

```diff
@@ -0,0 +1,70 @@
+---
+title: ElasticSearch Index Lifecycle Management
+owner: Services
+---
+
+<strong><%= modified_date %></strong>
+
+## <a id='overview'></a> Overview
+
+With the ElasticSearch index lifecycle management, you can create policies to eventually delete your old indexes. This is required, so your elasticsearch-service does not get filled with ancient data and eventually run into memory problems.
+
+## <a id='step-by-step-guide'></a> Step by Step Guide
+
+This section describes how to setup a basic index lifecycle management.
+
+### <a id='prerequisites'></a> Prerequisites
+
+To set up the index lifecycle management you already need ElasticSearch and Kibana working.
+- [ElasticSearch](./elasticsearch.html)
+- [Kibana Docker](./kibana-docker.html)
+
+### <a id='create-index-lifecycle-policy'></a> Create index lifecycle policy
+
+First of all, you need a index lifecycle policy. To create one in Kibana, go to 'Dev Tools'->Console. You may use this example or alter it according to your requirements.
+
+```txt
+PUT _ilm/policy/index-lifecycle
+{
+  "policy": {
+    "phases": {
+      "delete": {
+        "min_age": "60d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+```
+
+When the console acknowledges your request, the policy has been successfully created. This example deletes an index 60 days after it's been created.
+
+### <a id='create-index-template'></a> Create index template
+
+To create an index template, you can stay on the Console-page, we just used. You can use the following example.
+
+```txt
+PUT _template/logstash-template
+{
+  "index_patterns": ["index-*"],
```

Comment:
name of the index pattern and the match clause do not match the same. The name points to logstash indices but the match pattern matches for index-*